mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/SendScreenV2.tsx

```diff
@@ -520,14 +520,16 @@ function AddressRow ({
         },
         fieldState: { error }
       }) => {
-        const hasValidAddress = error?.type !== 'isValidAddress'
+        const hasValidAddress = error?.type == null
```

Comment:
This will help display the error, if there's any error. Meaning if the error is empty or error is not a valid address, then it will display the error message.